docs/source/apps/dfe/dfe_qss.ipynb

```diff
@@ -0,0 +1,592 @@
+    "    print(\"Installed qiskit-superstaq. Please restart the kernel to import the packages.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "39fbad63",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Required imports\n",
+    "import numpy as np\n",
+    "\n",
+    "# Optional imports\n",
+    "import os  # Used if setting a token as an environment variable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed311941",
+   "metadata": {},
+   "source": [
+    "To interface Superstaq via Qiskit, we must first instantiate a provider in `qiskit-superstaq` with `SuperstaqProvider()`. We then supply a Superstaq API token (or key) by either providing the API token as an argument of `qss.SuperstaqProvider()` or by setting it as an environment variable (see more details [here](https://superstaq.readthedocs.io/en/latest/get_started/basics/basics_qss.html#Set-up-access-to-Superstaq%E2%80%99s-API))."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "3181ee7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the qiskit superstaq provider for Superstaq backend\n",
+    "provider = qss.SuperstaqProvider()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac432c1c",
+   "metadata": {},
+   "source": [
+    "## Randomized measurements background"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a688217d",
+   "metadata": {},
+   "source": [
+    "The core idea behind this protocol is the use of random measurements to measure the overlap between two states $\\rho_1$ and $\\rho_2$, defined as $\\mathrm{Tr}(\\rho_1 \\rho_2)$. To do this, we measure both states in the same randomized Clifford product bases of the form $C_1 \\otimes \\cdots \\otimes C_N$ where each $C_i$ is a one qubit Clifford and we are comparing $N$-qubit (sub)systems. We then compare the distributions over all random measurements to get estimate the overlap between the two states.\n",
+    "\n",
+    "Succintly, the protocol goes as follows:\n",
+    "\n",
+    "1. Apply the same random unitary $U_j = C_1 \\otimes \\cdots \\otimes C_N$ to both states.\n",
+    "\n",
+    "2. Measure both states in the standard computational basis $\\left(|0\\rangle^{\\otimes N}, |1\\rangle^{\\otimes N}\\right)$.\n",
+    "\n",
+    "3. Repeat these measurementes for a fixed random basis to get an estimate of $P_{U_j}^i(x) = \\mathrm{Tr}\\left( U_j \\rho_i U_j^\\dagger |x\\rangle\\langle x| \\right)$ for all $x \\in \\{0, 1\\}^{\\otimes N}$.\n",
+    "\n",
+    "4. Repeat steps 1-3 for a collection of random unitaries $U = \\{U_1, \\cdots U_M\\}$.\n",
+    "\n",
+    "\n",
+    "With this data, we can calculate the overlap between the two states as:\n",
+    "\n",
+    "$$\n",
+    "\\mathrm{Tr}(\\rho_1 \\rho_2) = \\frac{2^N}{M} \\sum_{U_j \\in U} \\sum_{x, x' \\in \\{0, 1\\}^{\\otimes N}} (-2)^{-\\mathcal{D}(x, x')} P_{U_j}^1(x) P_{U_j}^2(x')\n",
```

Comment:
I think it would be a helpful supplement to briefly define/state that D(x,x') is the hamming distance and that M is the number of random unitaries (as an example).

Reply:
Added remarks defining those, lmk if you think that is enough.

Reply:
Thanks, that seems sufficient to me! For more information, the link to the paper is enough for now.